commons/src/test/java/com/simpletools/calculator/commons/FakeCalculatorActivity.kt

```diff
@@ -0,0 +1,28 @@
+package com.simpletools.calculator.commons
+
+import android.content.Context
+import com.simpletools.calculator.commons.activities.SimpleActivity
+import com.simpletools.calculator.commons.helpers.Calculator
+
+class FakeCalculator: SimpleActivity(), Calculator {
```

Comment:
What's the difference between this class and just the regular Calculator ?

Reply:
no activity behind it. basically I kept the method signatures to make it behave like a mock